pkg/shares/share_sequence_test.go

```diff
@@ -113,7 +113,7 @@ func Test_sparseSharesNeeded(t *testing.T) {
 		{appconsts.FirstSparseShareContentSize + appconsts.ContinuationCompactShareContentSize*99, 100},
 		{1000, 3},
 		{10000, 21},
-		{100000, 210},
+		{100000, 208},
```

Comment:
[Question/suggestion] Is there a specific formula that is used to derive the value of "208"? If such a formula exists, would it be possible to include a document here explaining the details of that formula? makes it easier to understand and maintain the test.